docs/en/seatunnel-engine/rest-api.md

```diff
@@ -262,3 +262,91 @@ network:
 
 </details>
 
+------------------------------------------------------------------------------------------
+
+### Encrypt Config.
+
+<details>
+<summary><code>POST</code> <code><b>/hazelcast/rest/maps/encrypt-config</b></code> <code>(Returns the encrypted config if config is encrypted successfully.)</code></summary>
+
+#### Body
+
+```json
+{
+    "env": {
+        "execution.parallelism": 1,
+        "shade.identifier":"base64"
```

Comment:
What other types of encryption do we support?  and base64 is not considered an encryption method and I can decrypt it easily.

Reply:
Base64 encryption and decryption are provided at now, and users can implement their own encryption and decryption.

Reply:
> Base64 encryption and decryption are provided at now, and users can implement their own encryption and decryption.

Can we tell users in the documentation how to extend the encryption method?

Reply:
Yes, we have documentation. [Config File Encryption And Decryption](https://seatunnel.apache.org/docs/2.3.3/connector-v2/Config-Encryption-Decryption)

Reply:
> Yes, we have documentation. [Config File Encryption And Decryption](https://seatunnel.apache.org/docs/2.3.3/connector-v2/Config-Encryption-Decryption)

Can you quote it in the document?  Users may not know how to customize encryption methods

Reply:
Yes, I will quote it tomorrow.